src/tests.rs

```diff
@@ -341,4 +412,29 @@ pub mod serverlesstest {
             "worker timed out\n\nCaused by:\n    deadline has elapsed"
         );
     }
+
+    #[actix_web::test]
+    async fn invalid_tx_hash_encoding_test() {
+        let app = test::init_service(new_app()).await;
+
+        let invalid_payload = json!({});
```

Comment:
use valid json body, to make sure test fails on invalid tx_hash.

Reply:
there's no valid body for an invalid hash anyway since there's no code to interpret it